.travis.yml

```diff
@@ -8,10 +19,8 @@ matrix:
       env: TOXENV=py33
     - python: "3.4"
       env: TOXENV=py34
-before_install:
-  - sudo apt-get update -qq
 install:
-  - sudo apt-get install -qq ldap-utils slapd
+  - pip install "pip>=7.1.0"
```

Comment:
No problem, it's part of the speed up. It's only a bit faster here, because pyldap doesn't have any pip-installed, compiled  dependencies - tox is pure python.